swap/tests/e2e.rs

```diff
@@ -26,15 +26,15 @@ async fn happy_path() {
             .await
             .unwrap();
 
-    let btc = bitcoin::Amount::from_sat(1_000_000);
+    let btc_to_swap = bitcoin::Amount::from_sat(1_000_000);
     let btc_alice = bitcoin::Amount::ZERO;
-    let btc_bob = btc * 10;
+    let btc_bob = btc_to_swap * 10;
 
     // this xmr value matches the logic of alice::calculate_amounts i.e. btc *
     // 10_000 * 100
-    let xmr = 1_000_000_000_000;
-    let xmr_alice = xmr * 10;
-    let xmr_bob = 0;
+    let xmr_to_swap = xmr_btc::monero::Amount::from_piconero(1_000_000_000_000);
+    let xmr_alice = xmr_to_swap * xmr_btc::monero::Amount::from_piconero(10);
```

Comment:
```suggestion
    let xmr_alice = xmr_to_swap * 10;
```
Multiplying amounts does not make sense.